docker/builder/cmd.sh

```diff
@@ -58,7 +58,8 @@ for SUITE_ITERATION in $(seq 1 "${MAX_TEST_SUITE_RETRIES}"); do
 
       set +e
       daemon -U --name sauce_connect -- ${SC_BINARY} \
-        -D *.ciscospark.com,*.wbx2.com,*.webex.com*,storage101.dfw1.clouddrive.com \
+        -D *.ciscospark.com,*.webex.com*,storage101.dfw1.clouddrive.com \
+        -t *.wbx2.com \
```

Comment:
so, after talking with the sauce folks, it's sounds like the solution may be to discard `-D` entirely, then use `-t internal-testing-services.wbx2.com,127.0.0.1`. According to the docs, anything not specified in -t will go straight through the internet and not through the tunnel.